test/services/GroupService.test.ts

```diff
@@ -0,0 +1,49 @@
+import GroupService from "../../src/services/GroupService";
+import GroupRepositoryService from "../../src/services/GroupRepositoryService";
+import spyOn = jest.spyOn;
+
+describe("GroupService", () => {
+    let groupService: GroupService | null;
+
+    beforeAll(() => {
+        GroupRepositoryService.prototype.getById = jest.fn();
+        GroupRepositoryService.prototype.createGroup = jest.fn();
+        GroupRepositoryService.prototype.updateGroup = jest.fn();
+        GroupRepositoryService.prototype.deleteGroup = jest.fn();
+        GroupRepositoryService.prototype.addUsersToGroup = jest.fn();
+
+        groupService = new GroupService(
+            new GroupRepositoryService(),
+        );
+    });
+
+    it("getById", () => {
+        const spy = spyOn(GroupRepositoryService.prototype, "getById");
+        groupService.getById("1");
```

Comment:
it's better to compare the result as well

Reply:
Did like this for speedup completion of this task. Is it a mandatory to add this now?